### PR TITLE
Update POM to reflect License & Authors

### PR DESCRIPTION
**Pull request by lwander**
Let me know if these changes are accurate.

### CODE REVIEW

File: kubernetes/pom.xml [hunk 1]
```diff
@@ -19,18 +19,18 @@
...
   </licenses>
 
   <developers>
     <developer>
-      <name>Swagger</name>
-      <email>apiteam@swagger.io</email>
-      <organization>Swagger</organization>
-      <organizationUrl>http://swagger.io</organizationUrl>
+      <name>The Kubernetes Authors</name>
+      <email>kubernetes-dev@googlegroups.com</email>
```
Comment (mbohlool) [inline kubernetes/pom.xml]: maybe kubernetes-sig-api-machinery@? @brendandburns What do you think?

Comment (brendandburns) [inline kubernetes/pom.xml]: Sure? I'm not sure it much matters, though I think eventually a sig-developers or sig-clients would be more appropriate than apimachinery (which seems dedicated to internals, not externals)

Comment (lwander) [inline kubernetes/pom.xml]: Is there an email address under the kubernetes.io domain that we could use perhaps?

**Comment by brendandburns** [comment]
I'm going to merge for now, @mbohlool we should perhaps bring up the topic of "contact emails" at the next community meeting...

**Pull request merged**